grr/server/grr_response_server/gui/ui/components/client_details/client_details.ng.html

```diff
@@ -22,274 +22,233 @@
   </div>
 
   <div *ngIf="timeline.selectedOptions.selected[0]?.value as client" class="info">
-    <div class="header">
-      <mat-icon>computer</mat-icon>
-      <div class="mat-h1 fqdn">{{client.knowledgeBase.fqdn}}
-        <div class="client-id mat-h3">{{client.clientId}}</div>
-      </div>
-    </div>
-    <div>
-      <div class="section-title mat-h3">General</div>
-      <table class="section">
+    <br/>
+    <div class="section-title mat-h3">OS Info</div>
+    <table class="section">
+      <tr *ngIf="client.osInfo.system">
+        <td class="description">System</td>
+        <td class="value">{{client.osInfo.system}}</td>
+      </tr>
```

Comment:
Not PR-related note: should we change this to "platform"?

Reply:
Users didn't point this out when we asked them about the content & format, so I wouldn't worry about it for now.